docs/manifest-types.md

```diff
@@ -110,6 +110,51 @@ This manifest adds a constant to specified target class. To set value you can sp
 }
 ```
 
+This manifest sets expression to specified method of a target class. To set a value you can specify a value field. It can be everything supported by PHP.
+
+```json
+{
+    "configure-module": [
+        {
+            "target": "\\Spryker\\Client\\Catalog\\CatalogConfig::targetMethod",
+            "value": 10
+        }
+    ]
+}
+```
+
+This manifest adds expression to specified method of target class. To set value you can specify value field. It can be everything supported by PHP.
+
+```json
+{
+    "configure-module": [
+        {
+            "target": "\\Spryker\\Client\\Catalog\\CatalogConfig::targetMethod",
+            "value": [
+                10
+            ]
+        }
+    ]
+}
+```
+
+This manifest substitutes expression of specified method target class in case if not exists any changes in according target method, checked by `previousValue` parameter. To set value you can specify value field. It can be everything supported by PHP.
+
+```json
+{
+    "configure-module": [
+        {
+            "previousValue": "$variable = 5 + 10; return $variable",
+            "target": "\\Spryker\\Client\\Catalog\\CatalogConfig::targetMethod",
+            "value": {
+                "value": "$variable = 10 + 20; $variable",
```

Comment:
I understand that this is an example, but I still don't get why we don't have a return statement here

Reply:
added return and in release app